ormar/models/mixins/save_mixin.py

```diff
@@ -402,3 +419,11 @@ def _get_field_values(self, name: str) -> List:
         if not isinstance(values, list):
             values = [values]
         return values
+
+    @classmethod
+    def get_fields_with_onupdate(cls) -> List[str]:
+        return [
```

Comment:
You didn't add a cached property like i.e. `_related_fields` in:

https://github.com/collerek/ormar/blob/8dc05d5cf48469f6285e88ab87d38b57c678976f/ormar/models/mixins/relation_mixin.py#L36-L53